TWLight/resources/admin.py

```diff
@@ -236,11 +236,12 @@ class SuggestionAdmin(admin.ModelAdmin):
 
 class PhabricatorTaskAdmin(admin.ModelAdmin):
     def partner_strings(self, object):
-        return [str(partner) for partner in object.partners.all()]
+        # return [str(partner) for partner in object.partners.all()]
+        return "Test"
 
     partner_strings.short_description = "partners"
 
-    search_fields = ("partner_strings", "phabricator_task")
+    search_fields = ("partners", "phabricator_task")
```

Comment:
I've just spotted this entirely unrelated test I was doing that I accidentally committed here.

Reply:
Removed this, squashed, and force pushed :)